### PR TITLE
WHO 5 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/everybody_lives!.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Name:Everybody Lives!
+ManaCost:1 W
+Types:Instant
+A:SP$ PumpAll | ValidCards$ Creature | KW$ Hexproof & Indestructible | SubAbility$ DBPump | SpellDescription$ All creatures gain hexproof and indestructible until end of turn. 
+SVar:DBPump:DB$ Pump | Defined$ Player | KW$ Hexproof | SubAbility$ DBEffect | SpellDescription$ Players gain hexproof until end of turn. 
+Players can't lose life this turn and players can't lose the game or win the game this turn.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/everybody_lives!.txt:6]: bad line

File: forge-gui/res/cardsfolder/upcoming/judoon_enforcers.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Name:Judoon Enforcers
+ManaCost:5 R W
+Types:Creature Alien Rhino Soldier
+PT:8/8
+K:Trample
+S:Mode$ AttackRestrict | MaxAttackers$ 1 | ValidDefender$ You | Description$ No more than one creature can attack you each combat.
+K:Suspend:6:1 R W
+Oracle:Trample\nSuspend 6—{1}{R}{W} (Rather than cast this card from your hand, you may pay {1}{R}{W} and exile it with six time counters on it. At the beginning of your upkeep, remove time counter. When the last is removed, cast it without paying its mana cost. It has haste.)
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/judoon_enforcers.txt:8]: incomplete

**Review by tool4ever — APPROVED** [review]

File: forge-gui/res/cardsfolder/upcoming/everybody_lives!.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+Name:Everybody Lives!
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/everybody_lives!.txt:1]: is that `!` in the filename ok? It makes me nervous 😅

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/everybody_lives!.txt:1]: yeah best to remove the ! to be sure

**Review by tool4ever — APPROVED** [review]

**Pull request merged**